cats/src/test/scala/frameless/cats/test.scala

```diff
@@ -83,13 +83,13 @@ class Test extends PropSpec with Matchers with PropertyChecks with SparkTests {
     toy.csum shouldBe theSeq.sum
   }
 
-  property("rdd Map[Int,Int] monoid example") {
-    import frameless.cats.implicits._
-    val rdd: RDD[Map[Int, Int]] = 1.to(20).zip(1.to(20)).toRdd.map(Map(_))
-    rdd.csum shouldBe 1.to(20).zip(1.to(20)).toMap
-  }
+  // property("rdd Map[Int,Int] monoid example") {
+  //   import frameless.cats.implicits._
```

Comment:
The `Map` instance was moved to `alleycats` in https://github.com/typelevel/cats/commit/1f0cba0490d22dc945a38f39c21f0a616202b574

Reply:
Hi @alonsodomin, can we bring this back based on @iravid suggestion?

Reply:
Hi @imarios, sorry for leavin this unattended. I just pushed an upgrade of the `cats-mtl` package.

There is still the problem with the `Map` test as there is no `CommutativeSemigroup` instance possible of an arbitrary `Map`, there could be one for `SortedMap` but `cats`(and `alleycats`) do not provide with one.

You can see @iravid comment below in which he acknowledges that he was thinking of the `Traversable` instance, not the `CommutativeSemigroup` one. So, in this case, if `frameless` is interested on enforcing correctness for those methods, probably it's better to drop the test for the `Map`... either that, or provide to `alleycats` a `CommutativeSemigroup` instance for `SortedMap` and wait until it's merged...

Reply:
What if we replace Map with SortedMap? Will we be able to have this test back?

Reply:
Hi @imarios, just made a PR to `cats` adding a `CommutativeMonoid` instance for `SortedMap` typelevel/cats#2047. Once it gets merged I can recover the commented out tests in here.